apps/src/codebridge/FileBrowser/FileBrowserHeaderPopUpButton.tsx

```diff
@@ -26,11 +33,13 @@ export const FileBrowserHeaderPopUpButton = () => {
   } = useCodebridgeContext();
   const {appName, validationFile} = levelProperties;
   const isBlockedAbuse = useAppSelector(state => state.lab.isBlockedAbuse);
+  const editableFileTypes =
```

Comment:
What do you think about having a map between app type and editable file types, rather than having ternaries like these? Maybe a problem for once we have three labs on codebridge :)

Reply:
https://codedotorg.atlassian.net/browse/AFL-146 - I wonder if whiteboard apps will be a Codebridge app.